package/aytests-tests.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jan 21 09:41:48 CET 2016 - schubi@suse.de
+
+- LVM: Add tests for value "auto" size paramter.
```

Comment:
`paramter` -> `parameter`

NP: Anyway, I'd write something like: `LVM: Add tests for value "auto" for "size" parameter`.